common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerData.kt

```diff
@@ -160,6 +169,18 @@ data class BrokerData(val packageName : String,
             AuthenticationConstants.Broker.LTW_APP_SHA512_DEBUG_SIGNATURE
         )
 
+        @JvmStatic
+        val accountManagerBrokers: Set<String> =
+            Collections.unmodifiableSet(object : HashSet<String>() {
```

Comment:
I believe this creates a new object on each invokation of the method? Could the (unmodifiable) list be declared as a constant in this file and then just return that constant?

Reply:
This is a static constant (val), not a method.

const in kotlin means compile-time constant. val = runtime constant.